lib/druid/client.rb

```diff
@@ -12,7 +14,14 @@ def initialize(broker_url, opts = nil)
     def send(query)
       uri = broker_uri(query)
 
-      req = Net::HTTP::Post.new(uri.path, {'Content-Type' =>'application/json'})
+      headers = {}
+      headers[X_REQUEST_ID] = RequestStore.store[:h4_request_id] if
+        RequestStore.store[:h4_request_id]
+      headers[X_ILB_REQUEST_ID] = RequestStore.store[:ilb_request_id] if
```

Comment:
The https://github.com/steveklabnik/request_store gem should be added https://github.com/TetrationAnalytics/ruby-druid/blob/master/Gemfile#L8

Reply:
sure 👍